apps/docs/components/react/getting-started.md

```diff
@@ -91,6 +94,21 @@ module.exports = {
 
 </SourceCode>
 
+### Modify Your Next.js Configuration File
+
+Storefront UI uses `react-use` package that needs additional transpilation step to be working with Next.js@13. To set this up, you need to add `transpilePackages: ['react-use']` into your `next.config.js` file:
```

Comment:
Can we have a call out that if someone's pre-Next 13, they can ignore this?

Reply:
talked over on slack - it's resolved already